clients/python/basic.py

```diff
@@ -0,0 +1,75 @@
+import math
+import signal
+import sys
+import time
+import logging
+from proton.handlers import MessagingHandler
+from proton.reactor import Container
+
+from config import Config
+from result import ResultData
+
+
+class BasicCommon(MessagingHandler):
```

Comment:
This class name is not so descriptive, is this a BasicWhat?

Reply:
The idea is that "Basic" is the "name" of the "basic clients", so we have "BasicSender" and "BasicReceiver". The "BasicCommon" is a common class for both Receiver and Sender.